docs/20-integration-guides/10-keycloak/03-create-realm.md

```diff
@@ -0,0 +1,52 @@
+# Create Realm and Client
+
+:::caution Keycloak version
+Based on your Keycloak version, steps to create realm and OIDC client may differ. Refer to the Keycloak documentation specific for your Keycloak version.
```

Comment:
Nestálo by za to uvést odkaz na dokumentaci? https://www.keycloak.org/docs/latest/server_admin/#assembly-managing-clients_server_administration_guide

Reply:
už vidím, že ona je odkazována o kus dále...